app/src/main/java/com/google/android/soundchecker/utils/AudioTrackSink.kt

```
@@ -66,7 +72,9 @@ class AudioTrackSink(var preferredDevice: AudioDeviceInfo? = null,
     }
 
     override fun start() {
-        open()
+        if (open() != AudioErrorCallback.SUCCESS) {
+            return
```

Are we failing silently?

---

When it fails, the caller knows via error callback.